<?php
include_once('../../includes/inc.php');
htmlHeader(array("title"=>"","metaDesc"=>"Dedicated to improve cancer outcomes via liquid biopsies.","preLoader"=>"0"));
?>
		<div class="main-container">
		<section class="page-title page-title-3 image-bg">
                        <div class="background-image-holder">
                            <img alt="Background Image" class="background-image" src="/img/comMut.jpg">
                        </div>
                        <div class="container">
                            <div class="row">
                                <div class="col-sm-12 text-center">
                                </div>
                            </div>

                        </div>

                        <ol class="breadcrumb breadcrumb-2">
                            <li><a href="#snv">SNVs</a></li>
                                <li><a href="#indel">Indels</a></li>
                            <li><a href="#fusion">Fusions</a></li>
                                <li><a href="#cnv">CNV</a></li>
                        </ol>
                    </section>
			<section>
				<div class="container">
					<div class="row">
					<div class="col-sm-12 text-center">
					    <h3 class="uppercase mb0">Comprehensive Variant Detection</h3>
						<h5 class="lead mb0">Hotspot Detection is not good enough.</h5>
					</div>
					<div class="row">
					<div class="col-sm-12 text-center">
						<p>The Resolution Bioscience ctDx platform has been validated to detect all four kinds of mutations. Other assays may only detect SNVs or small indels. Unfortunately, a cancer can progress with other types of mutations such as gene amplification. If your diagnostic test cannot detect all four kinds of mutations, then you may miss critical information required to make the proper clinical decision. 
						</p>
					</div>
					</div>
				    </div>
					<div class="row">
						<div class="col-md-6 col-md-push-3 text-center">
							<img alt="App" src="/img/mutationSlider.jpg">
						</div>
						
						<div class="col-md-3 col-md-pull-5">
							<div class="mt24 mt-xs-24 text-right text-left-xs">
								<h5 class="uppercase bold mb0">Single Nucleotide Variants</h5>
								<p class="fade-1-4">
									A SNV results when a single nucleotide changes from one base to another. 
								</p>
							</div>
							
							<div class="mt80 mt-xs-0 text-right text-left-xs">
								<h5 class="uppercase bold mb0">Indels</h5>
								<p class="fade-1-4">
									An insertion or deletion mutation (indel) is a type of genetic mutation where a sequence of nucleotides is inserted or deleted from the reference genome.
								</p>
							</div>
						</div>
						
						<div class="col-md-3 col-md-pull-1">
							<div class="mt24 mt-xs-0">
								<h5 class="uppercase bold mb8">Fusions</h5>
								<p class="fade-1-4">Hybrid gene formed by the joining of two separate genes through translocation, deletion, or inversion. 
								</p>
							</div>
							
							<div class="mt80 mt-xs-0">
								<h5 class="uppercase bold mb8">Copy Number Variation</h5>
								<p class="fade-1-4">
									Cells with CNV have an abnormal number of copies of one or more sections of the DNA.
								</p>
							</div>
						</div>
					</div>
					
				</div>
				
			</section>
			<section class="image-square left">
		        <div class="col-md-6 image"><a name="snv"></a>
		            <div class="background-image-holder">
		                <img alt="image" class="background-image" src="/img/squareSNV.jpg">
		            </div>
		        </div>
		        <div class="col-md-6 col-md-offset-1 content">
		            <h3>Single Nucleotide Variant</h3>
		            <p class="mb0">
				KRAS G12C (GGT>TGT) occurs in about 40% of NSCLC KRAS mutations.
		            </p>
				<h5 class="tableHead">KRAS G12C Clinical Plasma Specimen</h5>
				<table class="table table-striped table-condensed mutTable">
				<thead><th>Gene</th><th>Variant</th><th>AF</th><th>Location</th><th>Ref/Alt</th></thead>
				<tbody>
				<tr class="bold"><td>KRAS</td><td>G12C</td><td>1.5%</td><td>chr12: 25398285</td><td>C / A</td></tr>
				<tr><td>TP53</td><td>A159G</td><td>0.6%</td><td>chr17: 7578454</td><td>G / C</td></tr>
				</tbody>
				</table>
				A SNV can also cause a resistance mutation. EGFR T790M is an acquired resistance mutation to erlotinib and gefitinib.
				<h5 class="tableHead">EGFR T790M Clinical Plasma Specimen</h5>
                                <table class="table table-striped table-condensed mutTable">
                                <thead><th>Gene</th><th>Variant</th><th>AF</th><th>Location</th><th>Ref/Alt</th></thead>
                                <tbody>
                                <tr><td>EGFR</td><td>KELREA745K</td><td>30.7%</td><td>chr7: 55242464</td><td>AGGAATTAAGAGAAGC / A</td></tr>
                                <tr><td>TP53</td><td>S241F</td><td>15.2%</td><td>chr17: 7577559</td><td>G / A</td></tr>
				<tr class="bold"><td>EGFR</td><td>T7690M</td><td>3.1%</td><td>chr7: 55249071</td><td>C / T</td></tr>
                                </tbody>
                                </table>
		        </div>
		    </section><section class="image-square right">
		        <div class="col-md-6 image"><a name="indel"></a>
		            <div class="background-image-holder">
		                <img alt="image" class="background-image" src="/img/squareIndel.jpg">
		            </div>
		        </div>
		        <div class="col-md-6 content">
		            <h3>Insertions &amp; Deletions (Indels)</h3>
		            <p class="mb0">
				Indels are less common than SNVs. An indel in a coding region that is not a multiple of 3 nucleotides results in a frameshift mutation, which may alter the protein structure and function.
		            </p>
				<h5 class="tableHead">EGFR Indel Clinical Plasma Specimen</h5>
                                <table class="table table-striped table-condensed mutTable">
                                <thead><th>Gene</th><th>Variant</th><th>AF</th><th>Location</th><th>Ref/Alt</th></thead>
                                <tbody>
                                <tr class="bold"><td>EGFR</td><td>ELREAT746A</td><td>3.3%</td><td>chr7: 55242466</td><td>GAATTAAGAGAAGCAA / G</td></tr>
                                </tbody>
                                </table>
		        </div>
		    </section><section class="image-square left">
		        <div class="col-md-6 image"><a name="fusion"></a>
		            <div class="background-image-holder">
		                <img alt="image" class="background-image" src="/img/squareFusion.jpg">
		            </div>
		        </div>
		        <div class="col-md-6 col-md-offset-1 content">
		            <h3>Fusions</h3>
		            <p class="mb0">
				The ctDx platform is gene fusion partner and breakpoint agnostic &mdash; we discover novel fusions.
		            </p>
				<h5 class="tableHead">ALK Fusion Clinical Plasma Specimen</h5>
                                <table class="table table-striped table-condensed mutTable">
                                <thead><th>Gene</th><th>Variant</th><th>AF</th><th>Location</th></thead>
                                <tbody>
                                <tr class="bold"><td>ALK</td><td>EML4-ALK Fusion</td><td>0.53%</td><td>chr2:42552722-chr2:29446701</td></tr>
                                </tbody>
                                </table>
				<h5 class="tableHead">RET Fusion Clinical Plasma Specimen</h5>
				<table class="table table-striped table-condensed mutTable">
                                <thead><th>Gene</th><th>Variant</th><th>AF</th><th>Location</th></thead>
                                <tbody>
                                <tr class="bold"><td>RET</td><td>KIF5B-RET Fusion</td><td>16.26%</td><td>chr10:32315666-chr10:43610485</td></tr>
				<tr><td>TP53</td><td>R248G</td><td>8.5%</td><td>chr17: 7577539</td></tr>
				<tr><td>TP53</td><td>R181C</td><td>6.7%</td><td>chr17: 7578389</td></tr>
                                </tbody>
                                </table>
		        </div>
		    </section><section class="image-square right">
		        <div class="col-md-6 image"><a name="cnv"></a>
		            <div class="background-image-holder">
		                <img alt="image" class="background-image" src="/img/squareCNV.jpg">
		            </div>
		        </div>
		        <div class="col-md-6 content">
		            <h3>Copy Number Variation</h3>
		            <p class="mb0">
				The ctDx platform has been validated down to 0.5% ctDNA fraction for the detection of copy number amplifications. 
		            </p>

				<h5 class="tableHead">ERBB2 (HER2) CNV Clinical Plasma Specimen</h5>
                                <table class="table table-striped table-condensed mutTable">
				<thead><th>Gene</th><th>Variant</th><th>AF</th><th>Location</th><th>Ref/Alt</th></thead>
                                <tbody>
                                <tr class="bold"><td>ERBB2</td><td>Estimated 8.25 copies</td><td colspan="3"></td></tr>
				<tr><td>TP53</td><td>L257V</td><td>12.3%</td><td>chr17: 7577512</td><td>G / C</td></tr>
				<tr><td>TP53</td><td>M246MGGM</td><td>0.7%</td><td>chr17: 7577542</td><td>T / TCATGCCGCC</td></tr>
                                </tbody>
                                </table>

				<h5 class="tableHead">MET CNV Clinical Plasma Specimen</h5>
                                <table class="table table-striped table-condensed mutTable">
                                <thead><th>Gene</th><th>Variant</th><th>AF</th><th>Location</th><th>Ref/Alt</th></thead>
                                <tbody>
                                <tr class="bold"><td>MET</td><td>Estimated 19.9 copies</td><td colspan="3"></td></tr>
                                </tbody>
                                </table>

		        </div>
		    </section>
</div>					

<?php
footer();
?>
